src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Common/Notifications/Services/SmsHistoryService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AirBnB.Application.Common.Notifications.Services;
+using AirBnB.Persistence.Repositories.Interfaces;
+using FluentValidation;
+using System.Linq.Expressions;
+
+namespace AirBnB.Infrastructure.Common.Notifications.Services;
+
+public class SmsHistoryService(
+    ISmsHistoryRepository smsHistoryRepository,
+    IValidator<SmsHistory> smsHistoryValidator
+    ) : ISmsHistoryService
+{
+    public IQueryable<SmsHistory> Get(Expression<Func<SmsHistory, bool>>? predicate = null, bool asNoTracking = false)
+        => smsHistoryRepository.Get(predicate, asNoTracking);
+
+    public ValueTask<SmsHistory> CreateAsync(SmsHistory smsHistory, bool saveChanges = true, CancellationToken cancellationToken = default)
+    {
+        var validationResult = smsHistoryValidator.Validate(smsHistory);
```

Comment:
Use validation with IncludeRuleSets to specify the rule set to avoid the usage of default validation.